src/module.ts

```diff
@@ -1,24 +1,21 @@
-import { fileURLToPath } from 'url'
 import { defineNuxtModule, addPlugin, createResolver } from '@nuxt/kit'
 
-export interface ModuleOptions {
-  addPlugin: boolean
-}
+// Module options TypeScript inteface definition
+export interface ModuleOptions {}
 
 export default defineNuxtModule<ModuleOptions>({
   meta: {
     name: 'my-module',
     configKey: 'myModule'
   },
-  defaults: {
-    addPlugin: true
-  },
+  // Default configuration options of the Nuxt module
+  defaults: {},
   setup (options, nuxt) {
-    if (options.addPlugin) {
-      const { resolve } = createResolver(import.meta.url)
-      const runtimeDir = fileURLToPath(new URL('./runtime', import.meta.url))
-      nuxt.options.build.transpile.push(runtimeDir)
-      addPlugin(resolve(runtimeDir, 'plugin'))
-    }
+    const { resolve } = createResolver(import.meta.url)
```

Comment:
It would be better to keep `resolver` not destructured to allow accessing other utils and also explcitily to avoid conflict with other resolve fns such as one from path(e)

Reply:
Done in [62a36ad](https://github.com/nuxt/starter/pull/216/commits/62a36adf88f82ef04756b528cfa91c11bdc4eb46)